Documentation/ceph-object-bucket-notifications.md

```diff
@@ -0,0 +1,169 @@
+---
+title: Object Bucket Notification
```

Comment:
In the table of contents this might be the longest entry if we don't shorten it. How about this? There is enough context around it that I don't think we need "object" in the TOC title, and after they click on the topic they'll see the full title on line 7.
```suggestion
title: Bucket Notifications
```